packages/terra-functional-testing/src/terra-cli/wdio/index.js

```diff
@@ -32,30 +50,22 @@ const cli = {
         describe: 'A list of language locales for the test run.',
         default: ['en'],
       },
-      themes: {
-        type: 'array',
-        describe: 'A list of themes for the test run.',
-        default: ['terra-default-theme'],
-      },
-      hostname: {
+      site: {
         type: 'string',
-        describe: 'Automation driver host address.',
-      },
-      port: {
-        type: 'number',
-        describe: 'Automation driver port.',
+        describe: 'A file path to a static directory of assets. When defined, an express server will launch to serve the assets and disable running webpack.',
       },
-      baseUrl: {
-        type: 'string',
-        describe: 'The base url.',
+      spec: {
+        type: 'array',
+        describe: 'A list of spec file paths.',
       },
       suite: {
         type: 'array',
         describe: 'Overrides specs and runs only the defined suites.',
       },
-      spec: {
+      themes: {
```

Comment:
I'm starting to understand why the industry convention seems to be singular. I think it's fine to keep it plural, just an observation. I'm starting to see this more often:

```
    "wdio-lowlight": "terra wdio --themes clinical-lowlight-theme",
    "wdio-fusion": "terra wdio --themes orion-fusion-theme",
```

And Jenkins will end up looking like this after we eventually update it to use the cli:

`terra wdio --browsers chrome --formFactors huge --locales en`